### DISCUSSION TITLE
Preload Package Guids ?

### DISCUSSION THREAD

**Discussion by AnubisX9** [AssetRipper/AssetRipper#1491] [Q&A] [answered]
Is there a way that we could preload Package Guids ? Where in the project could we achieve something like this? 
Example: We build a blank project with all the required packages. We retrieve the packages from the Library folder. Then use those metadata files to marry the proper guids. The goal would be to do what this project is doing essentially but within AR.
https://github.com/oldsportsgit/GUIDcorrector (its very inefficient and slow) and Even with this I get issues loading prefabs (
Do not use ReadObjectThreaded on scene objects!) because guids are still wonky.

**Comment by ds5678** [comment, marked as answer]
That's an experimental feature in the premium edition.

**Reply by AnubisX9** [reply]
─────███────██
──────████───███
────────████──███
─────────████─█████
████████──█████████
████████████████████
████████████████████
█████████████████████
█████████████████████
█████████████████████
██─────██████████████
███────────█████████
█──█───────────████
█──────────────██
██──────────────█────────▄███████▄
██───███▄▄──▄▄███──────▄██$█████$██▄
██──█▀───▀███────█───▄██$█████████$██▄
██──█───█──██───█─█──█$█████████████$█
██──█──────██─────█──█████████████████
██──██────██▀█───█─────██████████████
─█───██████──▀████───────███████████
──────────────────█───────█████████
─────────────▀▀████──────███████████
────────────────█▀──────██───████▀─▀█
────────────────▀█──────█─────▀█▀───█
──▄▄▄▄▄▄▄────────██────█───████▀───██
─█████████████────▀█──█───███▀──▄▄██
─█▀██▀██▀████▀█████▀──█───██████▀─▀█
─█────────█▄─────────██───████▀───██
─██▄████▄──██────────██───██──▄▄▄██
──██▄▄▄▄▄██▀─────────██──█████▀───█
─────────███────────███████▄────███
────────███████─────█████████████
───────▄██████████████████████
████████─██████████████████
─────────██████████████
────────███████████
───────█████
──────████
─────████

ㅤ ㅤ ㅤ ㅤ ㅤ ▀▀█▀▀ㅤ█▀▀█ㅤ█ㅤ▄█ㅤ█▀▀ 
ㅤㅤ ㅤㅤ ㅤ ㅤㅤ█ ㅤㅤ█▄▄█ㅤ█▀▀▄ ㅤ█▀▀ 
ㅤ ㅤ ㅤ ㅤ ㅤ ㅤ ▀ㅤ ㅤ▀ㅤ ▀ㅤ▀ㅤㅤ▀ㅤ▀▀▀ 

█▀▄▀█ㅤ█ ㅤ █ㅤ ㅤ█▀▄▀█ ▄▀▀▄ █▀▀▄ █▀▀ █ ㅤ █ㅤ█ 
█─▀─█ㅤ█▄▄█ㅤㅤ █─▀─█ █ ㅤ █ █ ㅤ █ █▀▀ █▄▄█ㅤ█
█ㅤㅤ █ㅤ▄▄▄█ㅤ ㅤ█ㅤㅤ█ ─▀▀ ─ ▀ ㅤ ▀ ▀▀▀ ▄▄▄█ㅤ▄

**Reply by ds5678** [reply]
Let me know how it goes for you. This might help:
https://github.com/AssetRipper/MarrowMiningDemo

**Comment by dukemagus** [comment]
https://www.patreon.com/ds5678

Here's the Patreon link, just to be sure.

The differences between free and premium versions aren't written on the repo's main readme, by the way

**Reply by AnubisX9** [reply]
Cool, thank you. I signed up.
